extensions/wikia/AdEngine3/src/index.js

```diff
@@ -127,6 +127,17 @@ function trackTabId() {
   pageTracker.trackProp('tab_id', window.tabId);
 }
 
+function trackKruxSegments() {
+	const kruxUserSegments = context.get('targeting.ksg') || [];
+	const kruxICBMSegments = context.get('services.krux.icbmSegments') || [];
+
+	const kruxPropValue = kruxUserSegments.filter(segment => kruxICBMSegments.includes(segment));
+
+	if (kruxPropValue.length) {
```

Comment:
AC: `empty string if in no segments from list`. Something changed?

Reply:
We discussed that with @martynaut and the conclusion is not to send data to DW with empty string but only if there are segments.